ui/src/ducks/inventory/index.js

```diff
@@ -1,17 +1,91 @@
+import axios from 'axios'
 import { createAction, handleActions } from 'redux-actions'
+import { openSuccess } from '../alerts'
 
 const actions = {
   INVENTORY_GET_ALL: 'inventory/get_all',
+  INVENTORY_PENDING: 'inventory/pending',
+  INVENTORY_CREATE: 'inventory/create',
+  INVENTORY_UPDATE: 'inventory/update',
+  INVENTORY_DELETE: 'inventory/delete',
+  INVENTORY_REFRESH: 'inventory/refresh'
 }
 
 export let defaultState = {
-  all: []
+  all: [],
+  fetched: false,
 }
 
-export const findInventory = createAction(actions.INVENTORY_GET_ALL, () => {
-  //TODO
-})
+export const findInventory = createAction(actions.INVENTORY_GET_ALL, () =>
+  (dispatch, getState, config) => axios
+    .get(`${config.restAPIURL}/inventory`)
+    .then((suc) => {
+      openSuccess()
```

Comment:
- Look in the `ducks/alerts/index.js` file and observe the `defaultState`, `payload`, and `handleActions`. The `openSuccess()` should be describing its respective function's state (e.g., an alert should show saying "Retrieved all inventories" in `findInventory`).
- If `openSuccess()` occurs before we dispatch any action, did we do anything successful?